Formula/pgweb.rb

```diff
@@ -15,21 +15,15 @@ class Pgweb < Formula
   end
 
   depends_on "go" => :build
-  depends_on "go-bindata" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "auto"
-    (buildpath/"src/github.com/sosedoff/pgweb").install buildpath.children
+    ldflags = %W[
+      -s -w
+      -X github.com/sosedoff/pgweb/pkg/command.BuildTime=#{Date.today}
```

Comment:
We can either use `time` (but then it should wait for the next release of homebrew/brew (See https://github.com/Homebrew/brew/pull/11659 and https://github.com/Homebrew/homebrew-core/pull/80608) or `Time.at(ENV["SOURCE_DATE_EPOCH"].to_i).utc.iso8601` instead right now.

```suggestion
      -X github.com/sosedoff/pgweb/pkg/command.BuildTime=#{time.iso8601}
```

Reply:
I see, prety cool.